sdk/python/tests/compiler/testdata/artifact_location.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from kfp import dsl
+from kubernetes.client.models import V1SecretKeySelector
+
+
+@dsl.pipeline(name='foo', description='hello world')
+def foo_pipeline(tag: str, namespace: str = "kubeflow", bucket: str = "foobar"):
```

Comment:
Do you mind adding a sample similar with this one in the basic sample folder in another PR? Thanks.